src/globals.css

```
@@ -40,87 +40,105 @@
   --color-sidebar-accent-foreground: var(--sidebar-accent-foreground);
   --color-sidebar-border: var(--sidebar-border);
   --color-sidebar-ring: var(--sidebar-ring);
+  --font-sans: var(--font-inter-tight);
+  --font-serif: var(--font-newsreader);
+  --font-mono: var(--font-geist-mono);
 }
 /*---break---
```

Stylelint (17.10.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)